deploy.go

```diff
@@ -287,11 +289,11 @@ func isFailed(state string) bool {
 
 // waitState waits for a deployment status that matches the given states, then
 // sends on the returned channel.
-func waitState(states []string, owner, repo string, deploymentID int, c *github.Client) *github.DeploymentStatus {
+func waitState(states []string, owner, repo string, deploymentID int64, c *github.Client) *github.DeploymentStatus {
```

Comment:
Why the change here to `int64`? I'm assuming this gets deployed to a 64-bit machine so would be 64 bits anyway. I ask because the general best practice is to use the generic int type unless there's a specific reason to pivot.

Reply:
@arlandism I think that's just because the signature of `ListDeploymentStatueses` (used on 296) changed to `int64`

Reply:
Ah missed that